lizard_auth_server/views_api_v2.py

```diff
@@ -478,25 +477,31 @@ def form_valid(self, form):
                         user, portal)
 
         if not user:
+            # Try to find the user based on username
```

Comment:
Hier klopt het volgens mij niet. Deels cosmetisch, deels qua code.

De try/except/else, daar zit het eigenlijke dat je wilt doen in de 'else', dus in de uitzondering van de uitzondering. Een `if User....filter(...).exists():` geeft dan duidelijker weer wat je wilt.

Je zet de status_code, maar returnt hem niet. Dus de rest van de code loopt door. Dus een httpresponse met de juiste code returnen.

Reply:
Ja/nee, ik wil de user data met de 409 statuscode returnen, dus hergebruik de code daarvoor die al onderin de methode staat. Maar ben het met je eens dat het wat onduidelijk is zo, heb het een klein beetje omgegooid, denk dat het zo duidelijker is?

Reply:
Huh? 409 conflict, da's een error. Dus je hebt dan geen user data. Wat wil je dan terug sturen? Een niet bestaande foutieve user? :-)